edweiss-app/app/(app)/lectures/remotecontrol/index.tsx

```diff
@@ -44,6 +44,12 @@ const RemoteControlScreen: ApplicationRoute = () => {
     useEffect(() => { if (lectureDoc) { setPortrait(); } }, [lectureDoc]);
     useEffect(() => { updateSlideAudioRecording(talked, pageToTranscribe, courseName, lectureId, isRecording, currentPage, setPageToTranscribe, setTalked, () => startRecording(langCodeMap[lang])); }, [talked]);
 
+    // Function to set the current page
+    const setCurrentPageExternal = (page: number) => { setCurrentPage(page); };
+
+    // Expose the function to be used externally
+    (window as any).setCurrentPageExternal = setCurrentPageExternal;
```

Comment:
What is "window" ?

Reply:
Ah ha. You indeed have good eyes 👀.

So this is related to the issue #78. The main problem is that I need to set the current page from the `JumpToPageScreen` that get called by an `AbstractRmtCtl` component inside the `remotecontrol/index.tsx` file that hold the `currentPage` hook 😋.

To tackle this problem, many solution exists 🫣. I have detailed the one that workout for me and why did I choose this one. 

To keep things short, this allows me to access it outside so that I can set it up directly. You can see this a threat. But I made sure that the only one allowed to use it directly is the `handleGoToPage` that's in the `remotecontrol/utilsFunction`. Applying the fail safe default 😁.

Reply:
Thanks for the explanation! I see why you opted for this approach given the context of the currentPage hook and the need to set it from the JumpToPageScreen. The fail-safe mechanism with handleGoToPage sounds like a reasonable way to mitigate potential risks, though exposing window directly could still pose some challenges in terms of maintainability or unintended side effects down the line. Thus be careful when working with this. 

Other than that, I think it is fine to tackle your problem like this. Just keep in mind for any potential risks.

Reply:
Yep, exactly!